src/coreclr/vm/crst.h

```diff
@@ -90,12 +90,8 @@
 #include <minipal/mutex.h>
 
 #define ShutDown_Start                          0x00000001
-#define ShutDown_Finalize1                      0x00000002
 #define ShutDown_Finalize2                      0x00000004
```

Comment:
Rename now that we don't have `ShutDown_Finalize1`?

```suggestion
#define ShutDown_Finalize                       0x00000004
```